src/components/NcRichText/NcReferencePicker/NcProviderList.vue

```diff
@@ -80,9 +82,9 @@ export default {
 	},
 	methods: {
 		focus() {
-			this.$nextTick(() => {
-				this.$refs['provider-select']?.$el?.focus()
-			})
+			setTimeout(() => {
```

Comment:
Why a timeout? What happens if it takes longer?

Reply:
> Why a timeout?

Because it does not work when doing it on the next tick.

> What happens if it takes longer?

It fails to focus :grin: 

Do you know how focusing on the input element could be improved?

Reply:
If nextTick does not work, I don't have a better solution atm.